peeringdb_server/search_indexes.py

```diff
@@ -137,6 +137,7 @@ def get_model(self):
 
 class NetworkIndex(MainEntity, indexes.Indexable):
     org_id = indexes.IntegerField(indexed=False, model_attr="org_id")
+    asn = indexes.IntegerField(indexed=False, model_attr="asn")
```

Comment:
Quick check if this is used at all - looks like numeric asn searches are routed directly to django filters and are bypassing the indexed search entirely.

Reply:
I suspect this was a remnant of some experiments, and we didn't actually need to add this to the index.

Reply:
So we can remove it?

Reply:
Looking at this again, since it's `indexed=False` it does not actually add it to the search index, but instead makes it available as a property on the search result, which we need. So this is good as is :)